pkg/install/generate.go

```diff
@@ -47,6 +47,12 @@ const (
 
 	// UninstallJobLabel is the label used for counting the number of uninstall jobs in Hive
 	UninstallJobLabel = "hive.openshift.io/uninstall"
+
+	// ClusterDeploymentNameLabel is the label (along with ClusterDeploymentNamespaceLabel) that is used to identify the installer pod of a particular cluster deployment
+	ClusterDeploymentNameLabel = "hive.openshift.io/cluster-deployment-name"
+
+	// ClusterDeploymentNamespaceLabel is the label (along with ClusterDeploymentNameLabel) that is used to identify the installer pod of a particular cluster deployment
+	ClusterDeploymentNamespaceLabel = "hive.openshift.io/cluster-deployment-namespace"
```

Comment:
No need to track the namespace, we know it by looking at the pod itself.

Reply:
We need the namespace for watching the pod while reconciling the cd in selectorPodWatchHandler function. I am missing something?

Reply:
You can just use pod.Namespace, it's redudant information in the labels because the pod will always be in the same ns as the ClusterDeployment.